frontend/src/metabase/home/components/EmbedHomepage/FeedbackModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import {
+  Button,
+  Group,
+  Modal,
+  rem,
+  Stack,
+  Text,
+  Textarea,
+  TextInput,
+  Title,
+} from "metabase/ui";
+
+type FeedbackModalProps = {
+  opened: boolean;
+  onClose: () => void;
+  onSubmit: (feedback: { feedback: string; email: string }) => void;
+};
+
+export const FeedbackModal = ({
+  opened,
+  onClose,
+  onSubmit,
+}: FeedbackModalProps) => {
+  const [feedback, setFeedback] = useState("");
+  const [email, setEmail] = useState("");
+  const handleSubmit = () => onSubmit({ feedback, email });
+
+  return (
+    <Modal
+      size={rem(530)}
+      padding="xl"
+      opened={opened}
+      withCloseButton={false}
+      onClose={onClose}
+    >
+      <Title pb="sm">{t`How can we improve embedding?`}</Title>
+      <Stack spacing="lg">
+        {/* eslint-disable-next-line no-literal-metabase-strings -- only admins can see this component */}
+        <Text>{t`Please let us know what happened. We’re always looking for ways to improve Metabase.`}</Text>
+
+        <Textarea
+          label={t`Feedback`}
+          name="feedback"
+          placeholder={t`Tell us what happened`}
+          onChange={e => setFeedback(e.currentTarget.value)}
+          minRows={3}
+        />
+
+        <TextInput
+          label={t`Email`}
+          name="email"
+          placeholder={t`Leave your email if you want us to follow up with you`}
+          onChange={e => setEmail(e.currentTarget.value)}
+        />
+
+        <Group position="right">
+          <Button onClick={onClose}>{t`Cancel`}</Button>
+          <Button variant="filled" onClick={handleSubmit}>
+            {feedback.length > 0 ? t`Send` : t`Skip`}
```

Comment:
There could be some weird state here e.g. having an email with no feedback, but the button still say skip.
![image](https://github.com/metabase/metabase/assets/1937582/f0634105-db0c-4326-846f-89b502c78118)


I think in this case, we shouldn't force users to type the feedback since it will be a blocker. At least we get the email that we could follow up on. So I think the button should be `Send` at this point.
![image](https://github.com/metabase/metabase/assets/1937582/f3e1bb3a-4ed7-4ca1-ac30-0b01242d02fa)